db/seeds.rb

```diff
@@ -13,18 +13,20 @@
     puts employee.email
 end
 
-Kudo.create!(title: "sample title number 1", content: "sample content number 1", giver_id: 1, receiver_id: 5)
-Kudo.create!(title: "sample title number 2", content: "sample content number 2", giver_id: 1, receiver_id: 5)
-Kudo.create!(title: "sample title number 3", content: "sample content number 3", giver_id: 2, receiver_id: 4)
-Kudo.create!(title: "sample title number 4", content: "sample content number 4", giver_id: 2, receiver_id: 4)
-Kudo.create!(title: "sample title number 5", content: "sample content number 5", giver_id: 3, receiver_id: 4)
-Kudo.create!(title: "sample title number 6", content: "sample content number 6", giver_id: 3, receiver_id: 4)
-Kudo.create!(title: "sample title number 7", content: "sample content number 7", giver_id: 4, receiver_id: 3)
-Kudo.create!(title: "sample title number 8", content: "sample content number 8", giver_id: 4, receiver_id: 3)
-Kudo.create!(title: "sample title number 9", content: "sample content number 9", giver_id: 5, receiver_id: 1)
-Kudo.create!(title: "sample title number 10", content: "sample content number 10", giver_id: 5, receiver_id: 1)
+company_value1 = CompanyValue.create!(title:"Honesty")
```

Comment:
how about a bit more ruby way :)
```suggestion
%w[Honesty Ownership Accountability Passion].each do |company_value_title|
company_value1 = CompanyValue.create!(title:company_value_title)
```